guides/USE_JBDS.md

```diff
@@ -89,11 +89,11 @@ You can run the Arquillian tests that are included with many of the JBoss EAP 7.
 
 ## Known Issues for JBoss EAP Quickstarts
 
-You may see one or more of the following errors or warnings when you import quickstart projects into JBoss Developer Studio.
+You may see one or more of the following errors or warnings when you import quickstart projects into Red Hat CodeReady Studio.
 
 * **JRE System Library Problem: Build path specifies execution environment JavaSE-1.6. There are no JREs installed in the workspace that are strictly compatible with this environment.**
 
-    The JBoss EAP 6 quickstarts demonstrate Java EE 6 features and are built on Java SE 6. JBoss Developer Studio requires Java 8 to run, but is still capable of launching runtimes with various versions of Java. You can ignore this warning, or you can install a JDK 6 on your machine and add it to the Eclipse Java Runtime Environment by choosing `Preferences` --> `Java` --> `Installed JREs`.
+    The JBoss EAP 6 quickstarts demonstrate Java EE 6 features and are built on Java SE 6. Red Hat CodeReady Studio requires Java 8 to run, but is still capable of launching runtimes with various versions of Java. You can ignore this warning, or you can install a JDK 6 on your machine and add it to the Eclipse Java Runtime Environment by choosing `Preferences` --> `Java` --> `Installed JREs`.
```

Comment:
Should this be EAP 7 instead of 6?

Reply:
No, from what I see that's a particular issue that applies only to EAP 6